Sources/PayrailsCSE/PayrailsCSE.swift

```diff
@@ -152,21 +151,35 @@ public struct PayrailsCSE {
         
         return publicKeyRef
     }
+
+    private func extractPublicKey() throws -> String {
+        guard let cseConfig = cseConfig else {
+            throw NSError(domain: "ConfigError", code: 0, userInfo: [NSLocalizedDescriptionKey: "Missing Config"])
+        }
+        
+        if let tokenization = cseConfig.tokenization {
```

Comment:
I like this part where the logic is independent of the provider type. One question though - will there ever be a case where we have both?

Reply:
@kumaraksi Could be, but in that case shouldn't the publicKey be the same?